goodkey/good_key.go

```diff
@@ -43,11 +43,11 @@ type Config struct {
 	// WeakKeyFile is the path to a JSON file containing truncated modulus hashes
 	// of known weak RSA keys. If this config value is empty, then RSA modulus
 	// hash checking will be disabled.
-	WeakKeyFile string
+	WeakKeyFile string `validator:"required"`
```

Comment:
This field and BlockedKeyFile both document behaviors for when the field is empty, so they are not required.